rio_stac/stac.py

```diff
@@ -53,27 +56,105 @@ def get_metadata(
 
     metadata["bbox"] = list(bbox)
     metadata["footprint"] = bbox_to_geom(bbox)
+    return metadata
+
+
+def get_projection_info(
```

Comment:
reordered this section to split metadata retrieval into small independent function